src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return zeros`, () => {
```

Comment:
give more informative description: 'should return 0 for cat and dog when they are 14 years old' as en example. Change for other tests